packages/@react-aria/overlays/src/calculatePosition.ts

```diff
@@ -202,13 +203,16 @@ function computePosition(
   // add the crossOffset from props
   position[crossAxis] += crossOffset;
 
-  // this is button center position - the overlay size + half of the button to align bottom of overlay with button center
-  let minViablePosition = childOffset[crossAxis] + (childOffset[crossSize] / 2) - overlaySize[crossSize];
-  // this is button position of center, aligns top of overlay with button center
-  let maxViablePosition = childOffset[crossAxis] + (childOffset[crossSize] / 2);
+  // button bottom touching overlay bottom
+  let bottomsTouchPosition = childOffset[crossAxis] + childOffset[crossSize] - overlaySize[crossSize];
+  // button top touching overlay top
+  let topsTouchPosition = childOffset[crossAxis];
```

Comment:
Sure, here is how it was calculated before

max = childOffset[crossAxis] + (childOffset[crossSize] / 2) - overlaySize[crossSize]
![IMG_21C98B99B73F-1](https://user-images.githubusercontent.com/22986859/202410578-b723a51a-2fcb-4515-ab34-ebca9f8abe9a.jpeg)


min = childOffset[crossAxis] + (childOffset[crossSize] / 2)
![IMG_00EF142400AF-1](https://user-images.githubusercontent.com/22986859/202410593-dad1ff7a-dc4d-4005-a4e1-4989e0dc9a62.jpeg)

Reply:
This is how it is calculated after my changes

topsTouch = childOffset[crossAxis]
![IMG_8C5A6F6A8EE5-1](https://user-images.githubusercontent.com/22986859/202411295-a90b95d3-7c88-4431-83ae-8163361730f9.jpeg)

bottoms touch = childOffset[crossAxis] + childOffset[crossSize] - overlaySize[crossSize]
![IMG_12D976EC8B0B-1](https://user-images.githubusercontent.com/22986859/202411340-d8b6dc6f-e0ff-43d6-8152-a94a8e5fda09.jpeg)

Reply:
And we need to do the check at runtime which position is min and max because of this case
![IMG_73BC5A43416C-1](https://user-images.githubusercontent.com/22986859/202411831-da31da9b-aab0-453b-9326-b4d2b981bc29.jpeg)
![IMG_C024DCCBE216-1](https://user-images.githubusercontent.com/22986859/202411848-ef123573-ece0-4060-a5c6-d2e8f5a2aa95.jpeg)

The old solution was glued to the middle of the button if the button was at least 2 times bigger than the overlay